jupyter_server/gateway/gateway_client.py

```diff
@@ -598,18 +601,19 @@ def load_connection_args(self, **kwargs):
 
         # Give token renewal a shot at renewing the token
         prev_auth_token = self.auth_token
-        try:
-            self.auth_token = self.gateway_token_renewer.get_token(
-                self.auth_header_key, self.auth_scheme, self.auth_token
-            )
-        except Exception as ex:
-            self.log.error(
-                f"An exception occurred attempting to renew the "
-                f"Gateway authorization token using an instance of class "
-                f"'{self.gateway_token_renewer_class}'.  The request will "
-                f"proceed using the current token value.  Exception was: {ex}"
-            )
-            self.auth_token = prev_auth_token
+        if self.auth_token:
```

Comment:
#1339 looking at this issue. 

Is this the problematic line? Can we just revert this line?